IPython/html/base/handlers.py

```diff
@@ -250,6 +255,7 @@ def template_namespace(self):
             sys_info=sys_info,
             contents_js_source=self.contents_js_source,
             version_hash=self.version_hash,
+            **self.jinja_template_vars
```

Comment:
@ssanderson did you have an opinion on this? If you don't think there's a use for overrides, this can go in as-is.